lib/help_scout.rb

```diff
@@ -18,9 +20,14 @@ def initialize(api_key)
   def create_conversation(data)
     post("conversations", { body: data })
 
-    # Extract ID of created conversation from the Location header
-    conversation_uri = last_response.headers["location"]
-    conversation_uri.match(/(\d+)\.json$/)[1]
+    if last_response.code == 201
+      # Extract ID of created conversation from the Location header
+      conversation_uri = last_response.headers["location"]
+      return conversation_uri.match(/(\d+)\.json$/)[1]
+    elsif last_response.code == 400
+      # Validation failed so return the errors
+      raise HelpScout::ValidationError, last_response.parsed_response["message"]
+    end
```

Comment:
What about raising a catch all type thing here at the end if the response is not one of 201 or 400? I'm not super sure what kind of statuses they return at the moment though.